### PR TITLE
Russian l10n update

### PR DESCRIPTION
**Pull request by BychekRU**
@derjanb I've copied @IRainman's changes and fixed formating and translations, please merge and close #82

### CODE REVIEW

File: ru/messages.json [hunk 1]
```diff
@@ -110,12 +110,6 @@
     "Attention_Can_not_display_all_excludes_": {
         "message": "Внимание: список исключений был сокращен.\nПроверьте его вручную!"
     },
-    "Include_s__": {
```
Comment (derjanb) [inline ru/messages.json]: @BychekRU Thank you very much. One question: this removal is not intended, right?

Comment (BychekRU) [reply ru/messages.json]: @derjanb It's intended. It's about @include @exclude directives and shouldn't be localized I think.

Comment (derjanb) [reply ru/messages.json]: OK.

**Comment by derjanb** [comment]
Thank a lot!

**Pull request merged**